packages/integration-sdk-runtime/src/synchronization/__tests__/index.test.ts

```diff
@@ -59,6 +59,30 @@ describe('initiateSynchronization', () => {
     });
   });
 
+  test('starts a synchronization job with integration job id if provided', async () => {
+    const mockIntegrationJobId = 'test-integration-job-id';
+    const mockSyncJob = generateSynchronizationJob();
+    mockSyncJob.integrationJobId = mockIntegrationJobId;
+
+    const context = createTestContext();
+    context['integrationJobId'] = mockIntegrationJobId;
```

Comment:
NBD but just curious - any reason you're not using dot notation here?

Reply:
I can't use dot notation here because context comes from `createTestContext()`. That function does not return an actual `SynchronizationJobContext`, but an object that looks like one. Specifically, an object with three fields, one of which is _not_ integrationJobId.

That being said, the helper function for generating a sync job context should probably return an actual `SynchronizationJobContext`. I can update it so it does.

Reply:
> That being said, the helper function for generating a sync job context should probably return an actual SynchronizationJobContext. I can update it so it does.

That makes sense to me @JakeFerrero.

Reply:
Done.